### ISSUE TITLE
Removing entity from Chunk it's not in

### ISSUE THREAD

**Issue by ProfessorQu** [azalea-rs/azalea#135]
When an entity goes outside the render distance of the bot while another player is present it will occasionally trigger this warning message:
> WARN azalea_entity::plugin::indexing: Tried to remove entity 3v3 from chunk ChunkPos { x: 0, z: 0 } but the entity was not there.

The warning comes from [this line](https://github.com/azalea-rs/azalea/blob/7fc31f9506df8a0ad532741576473aedd13786b1/azalea-entity/src/plugin/indexing.rs#L174). It triggers in the cases of:
- A creature or player (including itself) dies.
- A creature or player goes outside the render distance (at least vertically as far as I've tested).

It's not a huge deal but I saw it and thought I'd see what I could come up with.

**Comment by SzczurekYT** [comment]
Actually it spams a lot of these.

**Comment by ProfessorQu** [comment]
Yeah it seems to scale with the number of entities, so in a normal world it spams it a bunch, but in a void world it just does it once for each entity that goes out of range or dies

**Comment by mat-1** [comment]
I can't reproduce this anymore, I think it was inadvertently fixed some time recently. I will note that sometimes Minecraft sends packets for entities that were never spawned (which caused warnings) and I'm not sure why (I can reproduce it by spawning a zombie and killing it near a bot). I've changed these to debug logs now though, so it shouldn't be a problem anymore.
